include/oneapi/tbb/concurrent_queue.h

```diff
@@ -131,8 +134,54 @@ class concurrent_queue {
         queue_allocator_traits::destroy(my_allocator, my_queue_representation);
         r1::cache_aligned_deallocate(my_queue_representation);
     }
+  
+    concurrent_queue& operator=( const concurrent_queue& other ){
+        //TODO: implement support for std::allocator_traits::propogate_on_container_copy_assignment
+        if (my_queue_representation != other.my_queue_representation) {
+            clear();
+	    my_queue_representation->assign(*other.my_queue_representation, my_allocator, copy_construct_item);
```

Comment:
```suggestion
            clear();
            my_allocator = other.my_allocator;
	    my_queue_representation->assign(*other.my_queue_representation, my_allocator, copy_construct_item);
```
Since we don't implement POCCA support now, it would be better to always copy assign allocators.